demoapp/src/main/java/me/uport/sdk/demoapp/request_flows/uPortLoginActivity.kt

```diff
@@ -36,11 +31,70 @@ class uPortLoginActivity : AppCompatActivity() {
         // create a DID
         val issuerDID = "did:ethr:${signer.getAddress()}"
 
+        /*@Suppress("StringLiteralDuplication")
+        val claim = mapOf(
```

Comment:
in a way, this way of building the request looks cleaner.
I would love it if we could come up with a DSL with the same look and feel but with proper type checks